### DISCUSSION TITLE
Sengled light bulb E11-N1EA missing Color and Temperature

### DISCUSSION THREAD

**Discussion by billyjoebob999** [Koenkk/zigbee2mqtt#13407] [Devices]
I just recently switched back to z2m and noticed that all my E11-N1EA bulbs no longer have Color or Temperature as an option. This worked just fine a few weeks ago. Tried deleting a bulb from z2m, resetting the device, and adding back to z2m but no change. I also have E21-N1EA and those have the Color/Temp options. Any suggestions?

**Comment by Koenkk** [comment]
See https://github.com/Koenkk/zigbee2mqtt/issues/13365

**Reply by billyjoebob999** [reply]
Hey, thank you for the info! Applied your workaround for the time being.
